tests/test_extensions/test_ext_intersphinx_cache.py

```diff
@@ -295,7 +295,7 @@ def test_load_mappings_cache_revert_update(tmp_path):
         app2.build()
         app2.cleanup()
 
-        # switch back to old url (re-use 'old_item')
+        # switch back to old url (reuse 'old_item')
```

Comment:
Likewise

Reply:
Any dictionary contain the word reuse, so the usual grammar rule is that you don't add the hyphen as far as I know. Following grammar rules makes it easier for now-native speakers like me. But then all variants are OK, I cerrtainly can live with re-use.

Reply:
Ok, I can be convinced on reuse -- let's leave as is in the PR

Reply:
I've already removed it. Besides, I see a preference for `re-use` among programmers nowadays, although `reuse` is more common in newspapers, books, etc. Languages evolve, there's always a gap between formal grammar rules and actual use, especially in English.

Reply:
Ok, thank you. Please avoid force-pushing, we squash-merge all PRs.